pandas/tests/test_window.py

```diff
@@ -1084,6 +1084,15 @@ def alt(x):
 
             self._check_moment_func(f, alt, name='quantile', quantile=q)
 
+            self.assertRaises(ValueError, mom.rolling_quantile,
+                              np.array([1, 2, 3]), window=3, quantile=-0.1)
```

Comment:
actually make this a separate function, and don't use an array, instead use the new format

``Series(....).rolling(3)``. We are dropping ``pd.rolling_quantile`` soon anyhow (as its been deprecated for a while).